packages/grafana-ui/src/components/TagsInput/TagsInput.tsx

```diff
@@ -28,7 +35,9 @@ export const TagsInput: FC<Props> = ({ placeholder = 'New tag (enter key to add)
 
   const onAdd = (event: React.MouseEvent) => {
     event.preventDefault();
-    onChange(tags.concat(newTagName));
+    if (!tags.includes(newTagName)) {
```

Comment:
@torkelo the idea here is not to add a tag if it's already in the list. Otherwise there's a weird situation when two same tags are in the list and when one of them is deleted, all disappear as they are matched by text. Don't think this will break anything, but wanted to double check anyways.

Reply:
looks good